spec/dummy/config/initializers/inflections.rb

```diff
@@ -11,6 +11,13 @@
 # end
 
 # These inflection rules are supported but not enabled by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
```

Comment:
You should add this to `config/initializers/inflections.rb` (outside of the dummy). Then it will be available to any app using Fae. Here, it'll only affect the dummy app.